dirsrvtests/tests/suites/config/compact_test.py

```diff
@@ -72,11 +73,31 @@ def test_compaction_interval_and_time(topo):
     """
 
     inst = topo.ms["supplier1"]
+
+    # Calculate the compaction time (1 minute from now)
```

Comment:
This should say 2 minutes, I'll fix it later.

Reply:
Could you, please, also, add copyright to this file?